lib/common/bitstream.h

```diff
@@ -162,7 +162,12 @@ MEM_STATIC size_t BIT_initCStream(BIT_CStream_t* bitC,
 FORCE_INLINE_TEMPLATE size_t BIT_getLowerBits(size_t bitContainer, U32 const nbBits)
```

Comment:
minor:

looking at the rest of the file,
it seems `bitContainer`'s type should rather be `BitContainerType`.

It doesn't change anything, since it's `typedef` to `size_t`, 
but it's just proper,
and could allow, in some future, to switch `BitContainerType` to employ more efficient `64-bit` registers even in `x32` mode.